report.md

```diff
@@ -475,3 +475,42 @@ Where is potential for improvement?
 What are your main take-aways from this project? What did you learn?
 
 Is there something special you want to mention here?
+
+
+## Statement of Contributions
+
+The following is a list of each group member's main responsibilities. There were some overlap
+and collaboration in several parts, so this is not exhaustive.
+
+- Katrina Liang
+  - Calculating the CCN of MaximumMinimumWindow::calculateMaxOfMin
+  - Measure the branch coverage of MaximumMinimumWindow::calculateMaxOfMin
+  - Add tests improving the branch coverage of MaximumMinimumWindow::calculateMaxOfMin
+  - Refactor MaximumMinimumWindow::calculateMaxOfMin  to reduce CC
+- Ludwig Kristoffersson
+  - Calculating the CCN of RedBlackBST::deleteFixup
+  - Measure the branch coverage of RedBlackBST::deleteFixup
+  - Add tests improving the branch coverage of RedBlackBST::deleteFixup
+  - Refactor RedBlackBST::deleteFixup to reduce CC
+- Marcus Alevärn
+   - Document the CCN of ten functions
+   - Creation of automated branch coverage report with Jacoco
+   - Writeup on our way of working
+   - Writeup on our overall experience
```

Comment:
Writeup?

Reply:
> _writeup_, a [report](https://dictionary.cambridge.org/dictionary/english/report) or [article](https://dictionary.cambridge.org/dictionary/english/article) that makes a [judgment](https://dictionary.cambridge.org/dictionary/english/judgment) about something

From https://dictionary.cambridge.org/dictionary/english/write-up

Could change to e.g. "document" if it is better